stdlib/websub/src/main/ballerina/websub/commons.bal

```
@@ -563,10 +563,16 @@ function WebSubHub::publishUpdate(string topic, string|xml|json|byte[]|io:Readab
 }
 
 function WebSubHub::registerTopic(string topic) returns error? {
+    if (!hubTopicRegistrationRequired) {
+        error e = { message: "Remote topic registration not allowed/not required at the Hub" };
```

Should we return this error?

---

ugh, yes. Fixed, thanks!